### PR TITLE
Course and review sorting with common sorting file

### PR DESCRIPTION
**Pull request by vn6**

### CODE REVIEW

File: tcf_website/static/common/sorting.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+function sortHTML(container, items, prop, asc=1){
+    cmp = cmpByProp(prop, asc);
+    var objs = $(items);
+    objs.detach().sort(cmp);
+    $(container).append(objs);
+}
+
+function cmpByProp(prop, asc) {
+    return function(a, b) {
+      id = '#'.concat(prop)
```
Comment (brian-yu) [inline tcf_website/static/common/sorting.js:16]: nit: semicolon

File: tcf_website/static/common/sorting.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      valA = a.querySelector(id).innerHTML.trim();
+      valB = b.querySelector(id).innerHTML.trim();
+
+      if(prop=='date'){ //convert date string to int
```
Comment (brian-yu) [inline tcf_website/static/common/sorting.js:20]: nit: bracket/paren spacing https://github.com/airbnb/javascript#whitespace

File: tcf_website/static/common/sorting.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        valA = dateToInt(valA);
+        valB = dateToInt(valB);
+      }else{ //extract number from string
```
Comment (brian-yu) [inline tcf_website/static/common/sorting.js:23]: spacing

File: tcf_website/static/common/sorting.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        return asc;
+      } else if (valA < valB) {
+        return -1*asc;
+      } else {
+        return 0;
+      }
+    }
+}
+
+function dateToInt(date_str){
```
Comment (brian-yu) [inline tcf_website/static/common/sorting.js:46]: spacing

**Comment by j-alicia-long** [comment]
@vn6 @jackliu612 merge the changes from dev and recommit and I'll give this a review :)

**Pull request merged**